### ISSUE TITLE
Starting proxy server from within editor or Rhino fails

### ISSUE THREAD

**Issue by tomvanmele** [compas-dev/compas#265]
It seems that from within an editor proxy servers only work if they can connect to an already existing server. If they have to start the server themselves, the process fails.

If a proxy server is launched from the command line first, everything works fine...

```
# on the command line
python -c "from compas.rpc import Proxy; Proxy()"
```

**Comment by tomvanmele** [comment]
this is solved since `bc73b0c`

**Comment by gonzalocasas** [comment]
bc73b0c
